app/src/main/res/layout-land/activity_info.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".InfoActivity"
+    android:background="@color/black"
+    android:id="@+id/root">
+
+<ImageView
+        android:id="@+id/ivPosterInfo"
+        android:layout_width="280dp"
+        android:layout_height="match_parent"
+        android:layout_marginStart="10dp"
+        android:layout_marginLeft="10dp"
+        app:srcCompat="@drawable/flicks_movie_placeholder"
+        tools:layout_editor_absoluteX="0dp"
+        tools:layout_editor_absoluteY="43dp" />
+
+    <TextView
+        android:id="@+id/tvTitleInfo"
+        android:layout_width="match_parent"
+        android:layout_height="43dp"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentEnd="true"
+        android:textStyle="bold"
+        android:layout_marginStart="10dp"
+        android:textColor="@color/white"
+        android:layout_marginTop="40dp"
+        android:layout_marginEnd="10dp"
+        android:layout_toEndOf="@+id/ivPosterInfo"
+        android:textAlignment="center"
+        android:textSize="24sp"
+        tools:text="Bloodshot" />
+
+    <RatingBar
+        android:id="@+id/ratingBar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tvTitleInfo"
+        android:layout_alignParentEnd="true"
+        android:layout_marginStart="75dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="90dp"
+        android:layout_toEndOf="@+id/ivPosterInfo"
+        android:progressTint="@color/orange"
+        android:secondaryProgressTint="@color/orange"
+        android:progressBackgroundTint="@color/grey"
+        android:numStars="5" />
+
+    <TextView
+        android:id="@+id/tvOverviewInfo"
+        android:layout_width="379dp"
```

Comment:
This might be too big to hardcode so it might not scale well for different devices